coalib/results/result_actions/PrintAspectAction.py

```diff
@@ -0,0 +1,17 @@
+from coalib.results.Result import Result
+from coalib.results.result_actions.ResultAction import ResultAction
+
+
+class PrintAspectAction(ResultAction):
+
+    @staticmethod
+    def is_applicable(result, original_file_dict, file_diff_dict):
+        return isinstance(result, Result) and (result.aspect is not None)
+
+    def apply(self, result, original_file_dict, file_diff_dict):
+        """
+        Print Aspect Information
+        """
+        print(type(result.aspect).__qualname__)
```

Comment:
just printing the qualname isn't really helpful, the main thing to do here is making use of all the great docs that the aspects have